e2e/errors_test.go

```diff
@@ -75,7 +75,6 @@ Error from server (AlreadyExists): error when creating "STDIN": deployments.apps
 		},
 	}
 	for _, tt := range tests {
-		tt := tt
```

Comment:
I think this is required to be done since we run tests in parallel.
let's rename the in loop variable `currentTT := tt`

Reply:
This is actually something that is a new feature in Golang 1.22, which makes the extra assigning not needed anymore. https://go.dev/blog/loopvar-preview has more details about it, as well as the **References to loop variables** section in the [Go 1.22 release notes](https://tip.golang.org/doc/go1.22).

Reply:
understood, That's awesome ! :D